### PR TITLE
Add minutes for last meeting

### PR DESCRIPTION
**Pull request by kierdavis**

### CODE REVIEW

File: 2019/meeting-2019-09-18.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# SR KTC Meeting 2019-09-18
+
+Date: Wednesday 18th September 2019, 19:30
+
+Present: Kier Davis, Rob Gilton, Dan Trickey, Tom Wheal
+
+## Agenda
+
+* Status of outstanding action points:
+    * TODO TODO TODO
```
Comment (trickeydan) [inline 2019/meeting-2019-09-18.md]: ?

File: 2019/meeting-2019-09-18.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    * Dan will investigate getting South Bank uni as a venue for brainstorming (#16)
+        * In progress
+* Kickstart kit prep
+    * Tom is happy to continue coordinating this
+    * Number of kits we want to support:
+        * Lots of stuff is in unknown state in the inventory, making it hard to discern how much of it is in a usable state
+        * Last competition year we initially shipped 37 kits, noting that we were cutting it very close on some components
+        * Maybe we should commit to a confident low number, and then increase later once we're more sure of the situation
+        * We'll probably need two events: one to review the current state of the kit and do any remaining testing/cleaning, and another to pack the kit boxes.
+* Provisional date for brainstorming: Sep 13
```
Comment (rgilton) [inline 2019/meeting-2019-09-18.md]: October 13th!

Comment (rgilton) [reply 2019/meeting-2019-09-18.md]: Also the venue is to be Southampton, as we'll all be around due to kit packing.

File: 2019/meeting-2019-09-18.md (hunk 1 above)
Comment (kierdavis) [reply 2019/meeting-2019-09-18.md]: Oops, didn't mean to commit that.

**Review by rgilton — APPROVED** [review]

**Pull request merged**